docs/src/snap/howto/security/hardening.md

```diff
@@ -0,0 +1,3429 @@
+# {{product}} hardening guide
+
+By default, {{product}} is designed to be compliant with all hardening
+guidelines outlined in the Comprehensive Checklist. These configurations ensure
+a secure and stable Kubernetes environment without requiring additional manual
+steps.
+
+However, some hardening configurations are not applied by default and require
+administrator intervention. These steps are outlined in the [Post-Deployment
+Configuration Steps section][]. These are omitted by default because they may:
+
+* Impact performance.
+*	Affect compatibility with workloads.
+*	Require input from the administrator.
+
+Please evaluate the implications of each configuration before applying it.
+
+## Post-Deployment Configuration Steps
+
+The following configurations, when applied, will enhance your Kubernetes
+cluster’s security posture. After completing these steps, your cluster will be
+fully compliant with both DISA STIG recommendations and CIS hardening
```

Comment:
Again fully compliant I would take to mean no warnings whatsoever. These checks will only remove failing tests